src/_layouts/base.njk

```diff
@@ -22,14 +22,14 @@ title: CS++
 
     <script defer data-domain="cspp.ie" src="https://plausible.io/js/script.js"></script>
 
-    <script src="https://cdn.jsdelivr.net/npm/ua-parser-js@1.0.37/src/ua-parser.min.js"></script>
+    <script async src="https://cdn.jsdelivr.net/npm/ua-parser-js@1.0.37/src/ua-parser.min.js"></script>
```

Comment:
Did you choose `async` over `defer` for a particular reason? https://stackoverflow.com/questions/10808109/script-tag-async-defer
![image](https://github.com/user-attachments/assets/4bd15ba9-fd5e-4a2a-ba49-a2281bc5c096)

Reply:
I already was changing it (See commit time for proof :P).
It has no impact on LCP as render delay remains 9.8s no matter what. I think this is to do with the CSS in how images grow to appear, etc.